src/main/java/com/sintern/service/StudentServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.sintern.service;
+
+import com.sintern.domain.Student;
+import com.sintern.repository.OpenInternPositionRepository;
+import com.sintern.repository.StudentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class StudentServiceImpl implements StudentService {
+
+    private StudentRepository studentRepository;
+    private EncryptionService encryptionService = new EncryptionServiceImpl();
```

Comment:
Use Spring for injecting dependencies. This is similar to how we inject a Repository into a Service.